README.md

```diff
@@ -49,3 +53,65 @@ To install FreeBSD-13.1 run the ```zfsinstall``` program with configuration para
 ##### Mirror disks ada0 and ada1
 
     zfsinstall -d ada0 -d ada1 -r mirror -s 4G -A -4 -c -p zroot
+
+### 5. Complete post-install actions
+Chroot to installed system.
+
+    chroot /mnt
+
+Create a group and username, set ssh keys.
+
+    pw groupadd YOUR-USER
+    pw useradd -m -n YOUR-USER -g YOUR-USER -G wheel -h - -c "your name"
+    cd /home/YOUR-USER
+    mkdir .ssh && chown YOUR-USER .ssh && chmod 700 .ssh
+    cd .ssh
+    vi authorized_keys    #paste in SSH pubkeys
+    chown YOUR-USER authorized_keys && chmod 600 authorized_keys
+    cd
+
+Configure `/etc/rc.conf` for hostname, networking, SSH server.
+
+A configuration suitable for Hetzner is listed below. Adapt to your settings:
+
+    vi /etc/rc.conf
+    
+    hostname="yourhostname"
+    ifconfig_igb0_name="untrusted"
+    ifconfig_untrusted="up"
+    ifconfig_untrusted_ipv6="up"
+    ifconfig_untrusted_aliases="inet 1.2.3.4/32 inet6 1234::123:123:1234::2/64"
+    ipv6_activate_all_interfaces="YES"
+    static_routes="gateway default"
+    route_gateway="-host 6.7.8.9 -interface untrusted"
+    route_default="default 6.7.8.9"
+    ipv6_defaultrouter="fe80::1%untrusted"
+    sshd_enable="YES"
+    zfs_enable="YES"
+
+Configure `/etc/resolv.conf` for DNS servers. Hetzner's are used in this example:
+
+    vi /etc/resolv.conf
+    
+    search YOURDOMAIN
+    nameserver 185.12.64.1
```

Comment:
I think their nameservers depend on the datacenter you’re in.

Maybe it would make sense to add script that can automate this task? (Network config)

Reply:
seems to be common ns for all hosted clients?

https://docs.hetzner.com/dns-console/dns/general/recursive-name-servers

Reply:
how to automate doing steps in a chroot? bunch of commands after chroot step?

Reply:
There are ways, but for what we need to do there, chroot isn’t really necessary (e.g., sysrc takes a parameter which file to change).

Reply:
need to add user & ssh keys inside chroot?

Reply:
`sysrc` supports `-f` to set the file to modify
`pw` supports `-V` to set an alternative etcdir
SSH keys are simply files we can copy around